src/newtask/NewTask.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import Form from "./Form";
+import NewTaskList from "./NewTaskList";
+
+const NewTask = () => {
+  const [toggleForm, setToggleForm] = useState(false);
+  const [newTask, setNewTask] = useState([]);
```

Comment:
since you're holding an array, it makes more sense to call this state `taskList` instead of `newTask`